src/app/parcels/add/[id]/page.tsx

```diff
@@ -1,17 +1,41 @@
 import { Metadata } from "next";
 import React from "react";
 import AddParcelForm from "@/app/parcels/add/AddParcelForm";
+import { Schema } from "@/database_utils";
+import supabase from "@/supabaseServer";
 
 interface AddParcelParameters {
     params: {
         id: string;
     };
 }
 
-const AddParcels: React.FC<AddParcelParameters> = ({ params }) => {
+export type CollectionCentresLabelsAndValues = [
+    Schema["collection_centres"]["name"],
```

Comment:
This doesn't seem right, does it need to be name then acronym

Reply:
No it has to be name and name
This is how the drop down menu labels and values work. The first element is the label displayed for user to see and the second one is the value stored in the useState. It is just coincidentally, the displayed label and the value stored this case is exactly the same
Acronym is not used in this file